### ISSUE TITLE
can we put signature before copy replying email?

### ISSUE THREAD

**Issue by kurokirasama** [ThomasHabets/cmdg#104]
When you respond to an email, first go your message, then the copy of the message you are replying to, and then the signature.

Is there a way to put the signature just after the new message? If not, it would be a cool addition ;)

**Comment by ThomasHabets** [comment]
That sounds like top-posting to me. Not top-posting is actually one of the selling points.

I've not added that feature since I personally don't want it. But I'd accept well-written PRs for it. Probably best implemented as a flag, for now.

**Comment by kurokirasama** [comment]
lol, I didn't even know that top-posting and bottom-posting were a thing... I've never even considered bottom-posting as a useful solution xD

**Comment by ThomasHabets** [comment]
@3ruce yeah not super intuitive how to do that. You create `~/.signature`, and start cmdg once with `-update_signature`. That uploads the signature to your Google Drive application specific directory (this way cmdg can store state in your Google Drive without being able to list, read, or write any "real" files).

After doing that once it's in the cloud, so will be retrieved automatically in the future.

This is so that you can run cmdg on multiple machines, and not need the signature configured on each one.
